app/views/settings/index.html.erb

```diff
@@ -139,6 +139,25 @@
       </div>
     <% end %>
 
+    <% if Keybase.enabled? %>
+      <div class="boxline">
+        <%= f.label :kb_username,
+          raw("<a href=\"https://keybase.io/\">Keybase</a>:"),
+          :class => "required" %>
+        <span>
+        <% if @edit_user.keybase_signatures.present? && Keybase.validate(@edit_user.keybase_signatures[0][:kb_username], @edit_user.keybase_signatures[0][:sig_hash], @edit_user.username) %>
```

Comment:
the comment is gone, but i agree that these requests are better done in the controller than in the view. i'd suggest setting up an instance variable in the controller (like `@edit_user`) for `@keybase_signatures` that's got everything on it. that way the view can just iterate through and show whatever it wants from it.